ci_framework/roles/edpm_build_images/tasks/package.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Package edpm-hardened-uefi image inside container image
+  args:
+    chdir: "{{ cifmw_edpm_build_images_basedir }}/tmp/edpm-image-builder"
+  ansible.builtin.shell:
+    cmd: |
+      buildah bud -f ./Containerfile.image -t edpm-hardened-uefi:latest > {{ cifmw_edpm_build_images_artifacts }}/edpm_hardened_uefi_container_build.log
```

Comment:
edpm_hardened_uefi_container_build.log and ironic_python_agent_container_build.log could go into role variables? they are also used in tasks/run.log

Reply:
@marios hey thanks :) it was my mistake it should be  edpm_hardened_uefi_container_package.log and not edpm_hardened_uefi_container_build.log , ironic_python_agent_container_package.log and not ironic_python_agent_container_build.log
corrected in follow PR.